benchmarks/bayesmark/report_template.md

```diff
@@ -4,14 +4,22 @@
 * Number of Models: {num_models}
 * Number of Datasets: {num_datasets}
 * Number of Problems: {num_problems}
+* Metrics Precedence: `{precedence}`
 
-Final score for each problem is calculated as `100 x (1-loss)`. Solver with highest score in each problem wins. For more details visit [bayesmark docs.](https://bayesmark.readthedocs.io/en/stable/scoring.html)
+Please refer to ["A Strategy for Ranking Optimizers using Multiple Criteria"][Dewancker, Ian, et al., 2016] for the ranking strategy used in this report.
+
+[Dewancker, Ian, et al., 2016]: http://proceedings.mlr.press/v64/dewancker_strategy_2016.pdf
 
 ## Table of Contents
 
-1. [Problem Leaderboards](#problem-leaderboards)
-2. [Datasets](#datasets)
-3. [Models](#models)
+1. [Overall Results](#overall-results)
+2. [Problem Leaderboards](#problem-leaderboards)
+3. [Datasets](#datasets)
+4. [Models](#models)
```

Comment:
> And, how about renaming `Problem Leaderboards` as `Individual Results` as `kurobako` does?

Done in 0da6fa1dfb3ba071c5036f415c23b99eedcb8c50.

> How about adding the "Solvers" and "Studies" sections as kurobako generated? 

Thanks for bringing this up. I was planning to emulate those sections as well (even provisioned bayesmark study IDs in partial reports), but figured it would blow up diff too much (required changes to `PartialReport` to retrieve and serve study IDs, `ReportBuilder` to render new sections and new templates for solver spec). Maybe it's better to leave this relatively big change as follow-up? I'll try my best to set some time aside for those outstanding tasks and actually finish them.

Reply:
> Maybe it's better to leave this relatively big change as follow-up?

Sounds reasonable. I agree with you.